src/NodaTime/Calendars/HebrewScripturalCalculator.cs

```diff
@@ -211,7 +213,7 @@ private static bool IsKislevShort(int year)
         internal static int ElapsedDays(int year)
         {
             int cache = GetOrPopulateCache(year);
-            return cache & ElapsedDaysCacheMask;
+            return cache >> 2;
```

Comment:
Wondered about it, but had thought it was a bit trivial. Given the comment, happy to do so :)